React/Views/RCTWrapperViewController.m

```diff
@@ -79,6 +79,17 @@ - (void)viewWillAppear:(BOOL)animated
       [bar setTitleTextAttributes:@{NSForegroundColorAttributeName : _navItem.titleTextColor}];
     }
 
+    if (_navItem.shadowHidden) {
+      // Remove the hairline shadow, but maintain translucency in the bar
+      for (UIView *view in bar.subviews) {
+        for (UIView *subview in view.subviews) {
+          if ([subview isKindOfClass:[UIImageView class]] && subview.frame.size.height < 2) {
+            [subview removeFromSuperview];
```

Comment:
hide it instead of removing it. we may want to support re-showing it if shadowHidden gets set back to false.